jsonrpc/jsonrpc.go

```diff
@@ -261,12 +271,6 @@ func (j *JSONRPC) handle(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	if req.Method == "GET" {
```

Comment:
Can we leave at least some GET endpoint as they are usually needed for default health-checks?   
Maybe we can have something like `/status` or `/health` that will return some data that will notify that the chain is running well.   

At the very least this health check endpoint should return 200 OK response.

Reply:
By the ethereum JSON-RPC spec, there are no `GET` endpoints.

We can use the `web3_clientVersion` endpoint as a health check.

Do you think it's better to add custom ones?

cc @Kourin1996

Reply:
Allow me to give you some feedback from what a user of the platform I developed told me. 
He is using the GET method as a health-check. BUT! In my opinion the health-check endpoint should not be in the RPC port, it should be served from the operations port (from where the Prometheus' metrics are exposed), call it `/healthz` as it is kind of the standard as far as I can tell. Returning text or a JSON object is up to the implementers as long as it returns status 200 (OK). For me as a Node operator it

Reply:
I love the idea honestly.

@ZeljkoBenovic ? What do you think? How should we expose a health ping endpoint?

Reply:
We should definitely keep GET `/health` , `/ping` or `/whatever` endpoint on `json-prc` API, as it can be used to quickly determine if the node is running or not.  

Especially useful in automated deployments, where you want to quickly check if the `json-rpc` API is running or not, which you can do with quick & dirty `curl` test ( instead of going with the whole POST `eth` endpoint shenanigans ) . 
And it really doesn't matter what it responds with as long as the status is 200 OK .

Reply:
@ZeljkoBenovic sorry to disagree. I think there should not be any endpoint using a pathname (`/health`) on the JSON-RPC port (doing it feels like implementing a REST endpoint). If you want to verify if the JSON-RPC port is working, you could use `web3_clientVersion` as @zivkovicmilos has said, but I would not use it as a constant health-check. 

Again, a health-check IMO should be part of the operations port, and its logic should check the status of the entire node, i.e. gRPC port, RPC port, more things if any.

Reply:
@inglkruiz 

We'll discuss internally as a team of adding a similar feature in the future 👍